dvclive/metrics.py

```diff
@@ -138,3 +142,9 @@ def read_step(self):
     def read_latest(self):
         with open(self.summary_path, "r") as fobj:
             return json.load(fobj)
+
+    @staticmethod
+    def _nested_set(d, keys, value):
+        for key in keys[:-1]:
+            d = d.setdefault(key, {})
+        d[keys[-1]] = value
```

Comment:
Should it be a part of logger? It seems to me like a dictionary manipulation method, we should probably get it out of the class as a separate function or even thing about utils package for the project. 

I like the `_nested_set` solution, thought if we will be tinkering more with `self.metrics` in the future, its worth remembering that we could add `dpath` as dependency which supports different nested operations with dict out of the box. But its probably not worh considering now, when we only need one operation.

Reply:
I opted for creating an `utils` module. `dpath` seems like a good choice if more operations are needed in the future